src/main/webapp/app/registry/config/config.component.scss

```diff
@@ -0,0 +1,11 @@
+/* ==========================================================================
+Spring cloud configuration page styles
+========================================================================== */
+
+.head {
```

Comment:
These classes are used in more than 1 page hence move these to `app.scss` and remove these files